### PR TITLE
fix(pypi): Use pipe to run a pypi dependency collector script

### LINKED ISSUE #348: [BUG] Unable to run full stack report for Python ecosystem on Windows
**Describe the bug**
 Unable to run full stack report for Python ecosystem on Windows.

**To Reproduce**
1. Run full stack report on any stack

**Expected behavior**
Dependency analytics report must be generated on Windows like any other OS.

**Logs**
```
'exec("
          ^
SyntaxError: EOL while scanning string literal
```

### PR DESCRIPTION
**Pull request by arajkumar**
Fixes #348.

### CODE REVIEW

File: src/statusMessages.ts [hunk 1]
```diff
@@ -16,6 +16,18 @@ export namespace StatusMessages {
   export const NO_SUPPORTED_MANIFEST = `No supported manifest's file found to be analyzed.`;
   export const PYPI_INTERPRETOR_PATH =
     'Provide path for python interpretor `Code/File -> Preferences -> Settings -> Workspace Settings`.For details check READMEs';
-  export const PYPI_INTERPRETOR_CMD = `-c 'exec("""\nimport pkg_resources as pr;import json,sys;gd=pr.get_distribution;res=list();\nfor i in open(sys.argv[1]):\n    try:\n        rs={};I=gd(i);rs["package"]=I.key;rs["version"]=I.version;rs["deps"]=set();\n        for j in pr.require(i):\n            for k in j.requires():\n                K=gd(k);rs["deps"].add((K.key, K.version))\n        rs["deps"]=[{"package":p,"version":v}for p,v in rs["deps"]];res.append(rs)\n    except: pass\na=sys.argv[2:3]\nop=open(a[0],"w")if a else sys.stdout\njson.dump(res,op)\n""")'`;
+  export const PYPI_INTERPRETOR_CMD = `
```
Comment (arajkumar) [inline src/statusMessages.ts]: There is no change in the logic of the script, I just beautified it for readability.

**Comment by miteshvp** [comment]
Build is not happy

**Comment by arajkumar** [comment]
@miteshvp It was a test issue, since the fix uses `spawn`, it has to be stubbed for the test.

**Comment by arajkumar** [comment]
[test]

**Comment by miteshvp** [comment]
@arajkumar - if you can add your comments on why tests were failing even after adding `spawn` stub.

**Comment by arajkumar** [comment]
@miteshvp, actually I had working `spawn` stub., but the current solution is simple & has less changes comparing to the one which uses `spawn`.

**Pull request merged**